src/EditorFeatures/Test/Diagnostics/SuppressMessageAttributeWorkspaceTests.cs

```diff
@@ -27,10 +30,39 @@ public class SuppressMessageAttributeWorkspaceTests : SuppressMessageAttributeTe
         protected override async Task VerifyAsync(string source, string language, DiagnosticAnalyzer[] analyzers, DiagnosticDescription[] expectedDiagnostics, string rootNamespace = null)
         {
             using var workspace = CreateWorkspaceFromFile(source, language, rootNamespace);
+
+            const string unconditionalSuppressMessageDef = @"
+namespace System.Diagnostics.CodeAnalysis
+{
+    [System.AttributeUsage(System.AttributeTargets.All, AllowMultiple=true, Inherited=false)]
+    public sealed class UnconditionalSuppressMessageAttribute : System.Attribute
+    {
+        public UnconditionalSuppressMessageAttribute(string category, string checkId)
+        {
+            Category = category;
+            CheckId = checkId;
+        }
+        public string Category { get; }
+        public string CheckId { get; }
+        public string Scope { get; set; }
+        public string Target { get; set; }
+        public string MessageId { get; set; }
+        public string Justification { get; set; }
+    }
+}";
+            var mscorlibRef = new[] { TestBase.MscorlibRef };
```

Comment:
>TestBase.MscorlibRef [](start = 38, length = 20)

How do we know this is the right version to use?

Reply:
I think this should be OK regardless: all 4.x+ corlibs should be unified, and I'm only using AttributeUsage, Attribute, and string. Those should always be present.

I looked at trying to find the references used by the workspaces layer, but the TestWorkspace was confusing and I couldn't figure out a good way to keep them in-sync.